src/Alert/Alert.tsx

```diff
@@ -55,6 +55,7 @@ export const Alert: React.FC<AlertProps> = ({
         [classes.warning]: variant === "warning",
         [classes.success]: variant === "success",
       })}
+      data-test="alert"
```

Comment:
It seems like we're mixing `data-test` `data-testid` and `data-test-id` across the project, we should stick with one and use it everywhere. I'd vote for `data-testid` as it seems to be the standard and it's more self-explanatory than `data-test`.